oracle/src/oracleimporter.cpp

```diff
@@ -66,6 +66,16 @@ bool OracleImporter::readSetsFromByteArray(const QByteArray &data)
     return true;
 }
 
+QString OracleImporter::getMainCardType(const QStringList &typeList)
```

Comment:
What will this do for "Artifact Creature" ? Reading it, it appears to put it with the Artifacts instead of the Creatures

Reply:
It loops following the order of items in `mainCardTypes`, so it will match Creature before Artifact

Reply:
Ahh, i was looking at the `.first()` return value and thought it was weird. Yeah, this makes sense :)